core/geotime/src/main/java/mil/nga/giat/geowave/core/geotime/index/dimension/TemporalBinningStrategy.java

```diff
@@ -211,6 +237,17 @@ public int getFixedBinIdSize() {
 				return StringUtils.stringToBinary((Integer.toString(value.get(Calendar.YEAR)) + "_"
 						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.MONTH)) + "_" + TWO_DIGIT_NUMBER.format(value
 						.get(Calendar.DAY_OF_MONTH))));
+			case HOUR:
+				return StringUtils.stringToBinary((Integer.toString(value.get(Calendar.YEAR)) + "_"
+						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.MONTH)) + "_"
+						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.DAY_OF_MONTH)) + "_" + TWO_DIGIT_NUMBER
+						.format(value.get(Calendar.HOUR_OF_DAY))));
+			case MINUTE:
+				return StringUtils.stringToBinary((Integer.toString(value.get(Calendar.YEAR)) + "_"
+						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.MONTH)) + "_"
+						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.DAY_OF_MONTH)) + "_"
+						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.HOUR_OF_DAY)) + "_" + TWO_DIGIT_NUMBER
+						.format(value.get(Calendar.MINUTE))));
```

Comment:
So what do we want to do here?  I can come up with an approach to compress these a bit, which will require me to rework the parsing logic within getStartEpoch(), and and adjust the fixed binId sizes.  Are we ok with this being a breaking change, or do we want to add complicated logic to handle both the new and old formats (a bit ugly)?